arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+{
+	"swagger": "2.0",
```

Comment:
Just took the swashbuckler output and formatted using http://jsonlint.com/ . Will have to figure out how I can get the right indentations automatically